ami/main/models.py

```diff
@@ -2288,21 +2379,25 @@ def list_names(self) -> str:
 
     def update_parents(self, save=True):
```

Comment:
If this Taxon parent changes, then all of its children need to be updated recursively

Reply:
@mihow Make a ticket for this issue